### PR TITLE
DAO article on wiki

### PR DESCRIPTION
**Pull request by AndrePanin**
added sections: developing-contracts/examples/DAO

### CODE REVIEW

File: docs/developing-contracts/examples/DAO.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+    dilution_bound: u128,
+    abort_window: u64,
+}
+```
+
+`admin` - the only member that is initially in DAO. He has 1 share that allows him to submit new proposal and therefore add new members.
+
+`approved_token_program_id` - the reference to the token contract (ERC20) that users use as pledge to get the share in the DAO.
+
+`period_duration` - the smallest unit time interval for the DAO.
```
Comment (xrdavies) [inline docs/developing-contracts/examples/DAO.md:30]: What is the unit here? ms or s

File: docs/developing-contracts/examples/DAO.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+`voting_period_length` - voting time interval.
```
Comment (xrdavies) [inline docs/developing-contracts/examples/DAO.md:32]: How about `Number of intervals for voting time`? Seems voting duration is `period_duration` * `voting_period_length`.

File: docs/developing-contracts/examples/DAO.md [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+    voting_period_length: u64,
+    grace_period_length: u64,
+    dilution_bound: u128,
+    abort_window: u64, 
+    total_shares: u128,
+    members: BTreeMap<ActorId, Member>,
+    member_by_delegate_key: BTreeMap<ActorId, ActorId>,
+    proposal_id: u128,
+    proposals: BTreeMap<u128, Proposal>,
+ 	approved_list: Vec<ActorId>,
```
Comment (xrdavies) [inline docs/developing-contracts/examples/DAO.md:54]: Seem there are some extra spaces in this line.

File: docs/developing-contracts/examples/DAO.md [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+
+```rust
+ pub struct Proposal {
+    pub proposer: ActorId, /// - the member who submitted the proposal
+    pub applicant: ActorId, /// - the applicant who wishes to become a member 
+    pub shares_requested: u128, /// - the amount of shares the applicant is requesting
+    pub yes_votes: u128, /// - the total number of YES votes for that proposal
+    pub no_votes: u128, /// - the total number of NO votes for that proposal
+    pub quorum: u128, /// - a certain threshold of YES votes in order for the proposal to pass
+  pub is_membership_proposal: bool, /// - true if it is a membership proposal, false if it is a funding 	 proposal
```
Comment (xrdavies) [inline docs/developing-contracts/examples/DAO.md:79]: format issue

File: docs/developing-contracts/examples/DAO.md [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+        guild_bank_id: &ActorId,
+        voting_period_length: u64,
+        period_duration: u64,
+        grace_period_length: u64,
+        abort_window: u64,
+        dilution_bound: u128,
+    )
+```
+
+#### Adds actors to approved list. The added actors can deposit their tokens to DAO and a DAO member can submit a proposal of joining them DAO.
```
Comment (xrdavies) [inline docs/developing-contracts/examples/DAO.md:118]: Better to add descriptions in the body, not in the title?

**Review by xrdavies — COMMENTED** [review]
Looks good, please find the comments above.

File: docs/developing-contracts/examples/DAO.md (hunk 1 above)
Comment (AndrePanin) [reply docs/developing-contracts/examples/DAO.md:30]: ms, 
added

File: docs/developing-contracts/examples/DAO.md (hunk 3 above)
Comment (AndrePanin) [reply docs/developing-contracts/examples/DAO.md:54]: fixed

File: docs/developing-contracts/examples/DAO.md (hunk 4 above)
Comment (AndrePanin) [reply docs/developing-contracts/examples/DAO.md:79]: fixed

File: docs/developing-contracts/examples/DAO.md (hunk 5 above)
Comment (AndrePanin) [reply docs/developing-contracts/examples/DAO.md:118]: changed

**Review by xrdavies — APPROVED** [review]
Looks good to me.

**Pull request merged**